destination/destination.go

```diff
@@ -0,0 +1,107 @@
+// Copyright © 2022 Meroxa, Inc.
```

Comment:
please check all the files for 2024 😄
```suggestion
// Copyright © 2024 Meroxa, Inc.
```